Automattic/jetpack
# Blaze: Enable in Jetpack by default

#### Changes proposed in this Pull Request:

Jetpack plugin: 
- This adds the Blaze functionality to the Jetpack plugin. It is enabled by default. 

Blaze package: 
- Adds initialization checks for loading the features.
- Adds Config package setup. 
- Updates readme. 
- Adds basic PHPunit tests

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






Test it in the Jetpack plugin: 
- Install this branch via the beta plugin or Jurassic link. 
- Connect Jetpack 
- Visit `/wp-admin/edit.php`
- Hover over the posts lists. You should see "promote" as a link, clicking on it should take you to the appropriate Blaze page. 

Test in wpcom/atomic
- Enable this branch on an atomic site using the beta plugin
- Publish a post. You should see the functionality of the post-editor button. 

Other: 
- Should not load any Blaze functionality when the user is not connected to wpcom.
